script.js

```diff
@@ -1,15 +1,35 @@
-let hash = location.href.split("#")[1];
+const search = location.search.substring(1); // Gets the location.search, and then removes the first character
 
-if (hash) {
-    let image = document.createElement('img');
+(function() {
+    if (!search) return; // if the search empty, then it stops.
```

Comment:
Maybe do this instead?
```js
if (search) {
    // Rest of the code
}
```
I'm not a huge fan of the anonymous function there..

Reply:
Ok, I could do that. But what do you have against anonymous functions?

Reply:
they make code look bad, not all, but most, see:
```js
// good
setInterval(() => {
    console.log('hi')
}, 300)


// bad
setInterval(function(){
    console.log('hi')
}, 300)
```